examples/device-dashboard/Makefile

```diff
@@ -39,10 +37,14 @@ packed_fs.c: $(wildcard web_root/*) Makefile web_root/main.css web_root/bundle.j
 	$(CC) ../../test/pack.c -o $(PACK)
 	$(PACK) $(wildcard web_root/*) > $@
 
-# Pull and build mbedTLS library. See https://mongoose.ws/tutorials/tls/#how-to-build for TLS build options
 mbedtls:
 	git clone --depth 1 -b v2.28.2 https://github.com/mbed-tls/mbedtls $@
-	$(MAKE) -C mbedtls/library
+
+ifeq ($(TLS), mbedtls)
+CFLAGS += -DMG_ENABLE_MBEDTLS=1 -Wno-conversion -Imbedtls/include
+CFLAGS += -DMBEDTLS_CONFIG_FILE=\"mbedtls_config.h\" mbedtls/library/*.c
+$(PROG): mbedtls
+endif
```

Comment:
`ifxx ($(CFLAGS_EXTRA), MBEDTLS)` ( if CFLAGS_EXTRA contains MBEDTLS)
so `make clean all CFLAGS_EXTRA= -DMG_ENABLE_MBEDTLS=1`just works

Reply:
I don't like this - snooping into CFLAGS_EXTRA